snapcraft/tests/test_commands_define.py

```diff
@@ -28,7 +28,7 @@ def test_defining_a_part_that_exists(self, mock_stdout):
         main.main(['define', 'curl'])
 
         expected_output = """Maintainer: 'none'
-Description: 'test entry for curl'
+Description: test entry for curl
```

Comment:
oh, but for search as that is the one that had the problem ;-)
This is still good!